gbrl/common/compression.py

```
@@ -0,0 +1,626 @@
+        targets = A @ V
+        actions = actions.to(self.device)
+        if log_std is not None:
+            log_std = log_std.to(self.device)
+
+        losses = []
+        if self.method == 'first_k' and not self.use_W:
+            return self.compression.get_parameters(A, V), [0]
+        # Guard: skip gradient loop if no optimizer
+        if self.optimizer is None:
+            return self.compression.get_parameters(A, V), [0]
+        for i in range(self.gradient_steps):
```

Ruff (0.14.14)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)